exchanges/stream/websocket_connection.go

```diff
@@ -228,18 +241,12 @@ func (w *WebsocketConnection) ReadMessage() Response {
 	case websocket.BinaryMessage:
 		standardMessage, err = w.parseBinaryResponse(resp)
 		if err != nil {
-			log.Errorf(log.WebsocketMgr,
-				"%v websocket connection: parseBinaryResponse error: %v",
-				w.ExchangeName,
-				err)
-			return Response{}
+			log.Errorf(log.WebsocketMgr, "%v websocket connection: parseBinaryResponse error: %v", w.ExchangeName, err)
+			return Response{Raw: []byte(``)} // Non-nil response to avoid the reader returning on this case.
```

Comment:
This seems like a patchup that could have unintended consequences. What situation caused this?

Reply:
No situation, I just noticed this would return the reader routine, the connection seems like it would still be valid, the websocket frame is intact so "should" 😆 be able to keep reading the next messages but continue to error if the binary response is garbage. I can roll this back if you so desire?

Reply:
I think its suitable enough